### ISSUE TITLE
SCs have lost their links to glossary terms

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#313]
### Current behavior

The Success Criteria have no links to glossary terms.

Example: [SC 1.1.1 in WCAG2ICT Editor's Draft](https://www.w3.org/TR/wcag2ict-22/#non-text-content):

> All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below.

### Expected behavior

The Success Criteria in WCAG2ICT should have links to glossary terms.

Example: [SC 1.1.1 in WCAG2ICT 2013](https://www.w3.org/TR/wcag2ict/#text-equiv-all):

> All [non-text content](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#non-text-contentdef) that is presented to the user has a [text alternative](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#text-altdef) that serves the equivalent purpose, except for the situations listed below.

Decision needed: Should all such links point to terms in the WCAG [Glossary](https://www.w3.org/TR/WCAG/#glossary) (as they did in WCAG2ICT 2013), or should some of these links point to terms in WCAG2ICT [Glossary items with specific guidance](https://www.w3.org/TR/wcag2ict-22/#glossary-items-with-specific-guidance)?

**Comment by maryjom** [comment]
@mitchellevan This is, in a sense, a duplicate of Issue #147. The solution to the broken links to terms wound up with a "fix" to remove all links from the WCAG included content.

This was unfortunately an intentional change. It is a result of the way the current document is built - which is different than before. Before (in 2013 WCAG2ICT), we used to copy the WCAG text into the document's source code manually and adjust all of the links to the terms to go to WCAG by changing the link to use the full URL.  Now we are including content from WCAG's GitHub, which uses internal document links rather than full URLS. This means they would point to the definitions in the WCAG2ICT document (not WCAG's), and be completely broken when WCAG2ICT does not include a specific definition with guidance in it; this happens with terms that apply to all technologies. There are also internal links in that content to things like the conformance information - yet more sections WCAG2ICT doesn't have. So if we want the WCAG term links, we'd have to find someone who can modify the scripting that is used to include the content from the WCAG repository and parse it and replace all internal document links with full URLs to the WCAG 2.2 Recommendation. Personally, I don't know how to do this and had relied on Michael Cooper to do this for me. We can see if Daniel can accomplish this change.

There are other things that don't work the same from the previous WCAG2ICT.  
- The notes original to each WCAG success criteria's content don't have the "Note" and numbering. (See issue #

**Comment by maryjom** [comment]
This issue has been fixed and is incorporated into the latest editor's draft.
